qcfractal/interface/collections/dataset_view.py

```diff
@@ -495,3 +498,90 @@ def _deserialize(data: bytes, msgpacked_cols: List[str]) -> pd.DataFrame:
             df.set_index("index", inplace=True)  # pandas.to_feather does not support indexes,
             # so we have to send indexless frames over the wire, and set the index here.
         return df
+
+
+class PlainTextView(DatasetView):
+    def __init__(self, path: Union[str, pathlib.Path]) -> None:
+        """
+        Parameters
+        ----------
+        path: Union[str, pathlib.Path]
+            File path of view
+        """
+        if isinstance(path, str):
+            path = pathlib.Path(path)
+        self._path = path
+
+    def write(self, ds: Dataset) -> None:
+
+        with tempfile.TemporaryDirectory() as tempd:
+            temppath = pathlib.Path(tempd)
+            mol_path = temppath / "molecules"
+            entry_path = temppath / "entries.csv"
+            value_path = temppath / "values.csv"
+            list_path = temppath / "value_descriptions.csv"
+            readme_path = temppath / "README"
+
+            mol_path.mkdir()
+            if "stoichiometry" in ds.data.history_keys:
+                molecules = ds.get_molecules(stoich=list(ds.valid_stoich),
+                                             force=True)  #TODO: change to function after pr 442
+            else:
+                molecules = ds.get_molecules(force=True)
+            for molecule in molecules['molecule']:
+                molecule.to_file(mol_path / f"{molecule.id}.xyz")
```

Comment:
Should we try to name these similar to the entry names? It would make it a bit more readable.

Reply:
Entry names are not great paths since they can contain spaces and other naughty characters. Additionally, names don't work well for reaction datasets with complicated stoichiometry. `entries.csv` provides a join table between entries and molecule ids.

Reply:
Does the phrase "join table" conflict with "familiar, broadly compatible, and inspect-able"? I agree that entry names are difficult (but can be converted), but are likely more readable. I wonder if we could consider a hybrid `entry_name.replace(" ", "_")... + stoich + mol_uid`?

Reply:
Good idea. I think I have done this with correct path normalization.